resources/Custom.stg

```diff
@@ -0,0 +1,47 @@
+main(ctx, definitions) ::= <<
```

Comment:
Add eProsima copyright and license to this file.

I have looked to the documentation PR and it does not seem to have any explanation of why the template has to be as this. Maybe some comments can be included in the template.

Reply:
Done in c0c98c5